assets/translations/en.json

```diff
@@ -122,6 +122,7 @@
     "followSystem": "Follow System",
     "obtainium": "Obtainium",
     "materialYou": "Material You",
+    "amoledTheme": "Pure black theme",
```

Comment:
Could you change this to "Use pure black dark theme" - this makes it more obvious that the toggle will only have an effect when the dark theme is on.

The key (`amoledTheme`) should also be closer to the value. Maybe `useBlackTheme`?

Also please add to all other language files (re-use the English version as a placeholder for others to update later).